### DISCUSSION TITLE
Video tutorial on Utsushi's Charm

### DISCUSSION THREAD

**Discussion by Yi-61** [chpoit/utsushis-charm#49] [General]
Hello, thanks for writing Utsushi's Charm and it is a life saver for MHRS players especially those on Switch! To help more Chinese-speaking players, I made a video tutorial on how to use Utsushi's Charm, including the installation, screen recording, text recognition and importing charms to armorset builders. The link to my video is: [https://www.bilibili.com/video/BV1fg411k7Le?p=3](https://www.bilibili.com/video/BV1fg411k7Le?p=3). 

I've noticed that there is already a [video guide in Chinese by Ippo Bnn](https://www.youtube.com/watch?v=YO2c2ktg9ws), but his video is in traditional Chinese and mine is in simplified Chinese. Furthermore, most people in mainland China do not have access to Youtube, but can easily access Bilibili where they discuss games and animes. So it would be great if my video tutorial could also be one of the references in this github repo.

Finally, we have written an armorset searcher website ([https://gamecat.fun/e](https://gamecat.fun/e)) that supports the charm files exported from Utsushi's Charm. We think our armorset searcher is very user-friendly and it even has qurioius augment auto-fill features that recommends practical armor qurious augments to users according to the desired skilled specified by users. We would appreciate it if our armorset searcher website could be listed here as another resource that supports Utsushi's Charm's export files.

Thank you!

**Comment by chpoit** [comment]
I added a link to your video and website, let me know how you want to be credited. I currently have you like this
`[bilibili] Simplified Chinese by Yi Liu (Gamecat)` in the readme.

I noticed your website can translate the english skills to chinese, would you mind if I took the skill list from your website and added it to the app? I currently only have the english skills for rise/sunbreak. It wouldn't make the chinese character recognition work, but it might help if I get time to add the in-app translation feature I've been wanting to add

Oh, and if you ever update your video for whatever reason, make sure to tell me and I'll update it.

**Reply by Yi-61** [reply]
Thank you! The video credit looks good. BTW, for the link to the GameCat armorset builder, you have to put the url as [https://gamecat.fun/e](https://gamecat.fun/e), and the "https://" cannot be ommitted otherwise Github thinks it's an internal link somehow. The current GameCat link does not work.

For the Chinese skill names, please feel free to take the skill list from GameCat (would be great if you could just cite GameCat Wiki for this). We can also send you a json file for the translation so that you don't have to manually copy paste from the website, if that's easier for you. Just let us know if you need this!

And yes, will keep you in the loop if we have new video updates.

**Reply by Yi-61** [reply]
Actually, could you credit me like this:
[bilibili] Simplified Chinese by Gamecat
Thank you so much! : )

**Reply by chpoit** [reply]
Done
